packages/query/src/block-processor.ts

```diff
@@ -162,12 +162,12 @@ export class BlockProcessor implements BlockProcessorInterface {
 
     // special case genesis sync
     if (currentHeight === -1n) {
-      // initialize validator info at genesis
-      // TODO: use batch endpoint https://github.com/penumbra-zone/penumbra/issues/4688
```

Comment:
suggestion: think it would be helpful to keep this comment

Reply:
added